### ISSUE TITLE
Dividers showing the text 'Divider' instead of an actual divider.

### ISSUE THREAD

**Issue by pattonwebz** [wp-bootstrap/wp-bootstrap-navwalker#154]
Placing "Divider" or "divider" as the title attribute or the link text of a custom menu item won't add a divider to the menu. It's just adding a menu item with the text of "Divider".

Entering this:

![menus gaming now wordpress](https://cloud.githubusercontent.com/assets/3902039/5643741/ed056e06-9653-11e4-8c95-032f28971907.png)

Results in this:

![assassin s creed iii art of the assassin auction for rock the vote](https://cloud.githubusercontent.com/assets/3902039/5643777/67f9e844-9654-11e4-9184-9182bfa4c265.png)

Tried using both 'divider' and 'Divider' in label and title fields. Leaving the label field empty but filling in title results in no output.

**Comment by dahlskebank** [comment]
If this regarding bucaran's version, it is missing an if statement for divider.

```
if ('divider' === strtolower($item->attr_title) && $depth === 1) {
  return $indent . '<li role="presentation" class="divider">';
}
```

I think I added this after line 165, although I have no idea how to use github, so I may be referencing the wrong file. However this gave me an unwanted side effect, empty <li> tags.

![image](https://cloud.githubusercontent.com/assets/7286491/5658756/bc3266b2-9707-11e4-8170-82b2975c6b1f.png)

Seeing as how I can't code for shit, I'm waiting for someone else to fix this. There's only so much copy+paste can do...

**Comment by pattonwebz** [comment]
Hey @dahlskebank this issue isn't regarding the version bucaran submitted - this issue affects the original version currently in the master branch.

I'm gonna try fix this one myself and make a merge request for it on the master to fix the dividers. Should port over pretty much the same to bucaran's version as well once I figure it out.

**Comment by dahlskebank** [comment]
@pattonwebz: I finally understood this problem... You mean when you use "divider" as a main/parent menu item. That will never work properly, as it is not part of the Bootstrap component. I can't find any vertical divider. Maybe I missed it?

Link for reference:
http://getbootstrap.com/components/#dropdowns-divider

I believe it has to be used as a sub item, because the Bootstrap divider class is horizontal only, and by the looks of it, divider requires ul to have the class dropdown-menu.

**Comment by pattonwebz** [comment]
@dahlskebank Ahh yeah you figured it out - only horizontal dividers for sub-items are provided by BS3. 

I placed the "Divider" inside a dropdown to make it a sub-item it shows as I expected.

I appreciate all your help, Daniel. Without you I probably would never have noticed BS3 didn't support the vertical dividers.

In the old BS2 there was vertical dividers as well (with a horizontal mode in the responsive version - yes responsive used to be an optional extra in those days lol) for top level items. http://getbootstrap.com/2.3.2/components.html#navbar Lower box in the "Nav Links" section shows those.

Closing this issue as it's not actually a bug.
